src/helpers/mobileApi.js

```diff
@@ -0,0 +1,40 @@
+const qs = require('qs')
+const fetch = require('node-fetch')
+
+const baseUrl = process.env.MOBILE_API_BASE_URL
+const apiKey = process.env.MOBILE_API_TOKEN
+
+const makeRequest = async (route, params) => {
+  const urlRoute = [baseUrl, route].join('/')
+  let url = urlRoute
+  if (params) {
+    url = [urlRoute, qs.stringify(params)].join('?')
+  }
+
+  const response = await fetch(url, {
+    method: 'GET',
+    headers: {
+      'Authorization': `Basic ${apiKey}`,
+      'Cache-Control': 'no-cache',
+      'Content-Type': 'application/json',
+    },
+  })
+
+  if (!response.ok) {
+    throw { status: response.status, message: response.statusText }
+  }
+
+  const text = await response.text()
```

Comment:
Can this `await response.text()` throw too?

Reply:
I copied this from the `cellHotspots.js` file you wrote 3 months ago, so I'm not sure, can it? 😆

Reply:
Ah ok I think I got that mostly from one of the other ones we have haha. Seems like its been working fine, so I would say its good to go.